src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,4 +1,5 @@
-<div id="sfwdt{{ token }}" class="sf-toolbar sf-display-none" role="region" aria-label="Symfony Web Debug Toolbar">
+<!-- START of Symfony Web Debug Toolbar -->
+<div id="sfwdt{{ token }}" class="sf-toolbar sf-toolbar-closed" role="region" aria-label="Symfony Web Debug Toolbar">
```

Comment:
showing the closed state during loading time is not equivalent to the existing behavior, where we see *nothing* until the toolbar is loaded.